src/services/backend.js

```diff
@@ -8,6 +8,6 @@ if (!Backend) {
     'module specified by CONNECTOR_BACKEND was not found in /backends')
 }
 
-module.exports = new Backend({
+module.exports = new Backend(config, {
```

Comment:
Instead of passing the entire config, can you pass in just the required config values, similar to how fxSpread is being passed in?